logstash-plugins/usagetype-plugin/src/test/java/org/reactome/logstash/plugins/usagetype/TestIPUtils.java

```diff
@@ -0,0 +1,94 @@
+package org.reactome.logstash.plugins.usagetype;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.math.BigInteger;
+import java.net.UnknownHostException;
+
+import org.junit.Test;
+import org.reactome.logstash.plugins.usagetype.util.IPUtils;
+
+@SuppressWarnings("static-method")
+public class TestIPUtils
+{
+	@Test
+	public void testConvertIPToNumber() throws UnknownHostException
+	{
+		String ipString = "192.168.10.1";
+		BigInteger ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(3232238081l, ipNum.longValue());
+
+		ipString = "0.0.0.0";
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(0l, ipNum.longValue());
+
+		ipString = "255.255.255.255";
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(4294967295l, ipNum.longValue());
+
+		ipString = "172.217.165.14"; // This one is Google!
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(2899944718l, ipNum.longValue());
+
+		ipString = "127.0.0.1";
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(2130706433l, ipNum.longValue());
+
+		ipString = "206.108.127.16";
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertEquals(3463216912l, ipNum.longValue());
+
+		// Test some IPv6 values.
+		ipString = "2001:4860:4860::8888";
+		ipNum = IPUtils.convertIPToNumber(ipString);
+		assertTrue("42541956123769884636017138956568135816".equals(ipNum.toString()));
```

Comment:
Could the first parameter in the assert statements be given named constants?  Otherwise, they seem like magic numbers.

Reply:
They are the long-forms of the IP addresses. I am not sure a big bunch of named constants is worth it in this case.